packages/tradershub/src/components/DemoRealSwitcher/DemoRealSwitcher.tsx

```diff
@@ -0,0 +1,78 @@
+// TODO: Create a Generic component for this once the designed is finalized.
+import React, { useState } from 'react';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { LabelPairedChevronDownSmRegularIcon } from '@deriv/quill-icons';
+import { Listbox, Transition } from '@headlessui/react';
+
+const accountTypes = [
+    { label: 'Demo', value: 'demo' },
+    { label: 'Real', value: 'real' },
+];
+
+const DemoRealSwitcher = () => {
+    const [selected, setSelected] = useState(accountTypes[0]);
+    const { label, value } = selected;
+
+    return (
+        <div className='w-auto'>
+            <Listbox as='div' onChange={setSelected} value={selected}>
+                {({ open }) => (
+                    <div className='relative mt-1'>
```

Comment:
do we have mt-1 ?

Reply:
Nope! Thanks for pointing it out will replace it 🙇🏼